modules/data_release/test/TestPlan.md

```diff
@@ -0,0 +1,40 @@
+# Data Release Module Test Plan 
+
+Ensure that regular users can see the data release module without the 
+'Upload File' and 'Add Permission' buttons at the top.
+
+
+## Upload File
+1. Give the user the `data_release_upload` *(Data Release: Upload file)* permission. 
+ Ensure the user can upload new files to the module
+ 
+2. As the 'superuser', click on the 'Upload File' button to upload a file 
+ and ensure the uploaded file is available in the list of files
+ 
+## Permissions
+
+1. Give the user the `data_release_edit_permissions` *(Data Release: Give user 
+permission to view files)* permission.
+
+2. Click on the 'Add Permission' button.
+
+3. Select another 'User ID' (that you have access to and is not a 'superuser').
```

Comment:
`User ID` I would use only the word user. Same for `Data Release ID` below, I would use a user with `data release` permission.

Reply:
@PapillonMcGill I addressed all ur comments and added some content as well

please re-review